tests/h/services/annotation_json_presentation_test/_formatters/flag_test.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+
+from h.services.annotation_json_presentation._formatters.flag import FlagFormatter
+
+
+class TestFlagFormatter:
```

Comment:
This is another total rewrite. The previous version of the tests used the real service which made it a total pain to adapt.